srcpkgs/GammaRay/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'GammaRay'
+pkgname=GammaRay
+version=2.11.1
+revision=1
+wrksrc="gammaray-${version}"
+build_style=cmake
+configure_args="-DGAMMARAY_INSTALL_QT_LAYOUT=true -DECM_MKSPECS_INSTALL_DIR=/usr/lib/qt5/mkspecs/modules -DPLUGIN_INSTALL_DIR=/usr/lib/qt5/plugins/gammaray"
+hostmakedepends="qt5-host-tools qt5-devel"
+makedepends="qt5-devel"
+short_desc="Tool to poke around in a Qt-application"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/KDAB/GammaRay"
+distfiles="${homepage}/releases/download/v${version}/gammaray-${version}.tar.gz"
+checksum=87a1d72ad1ad6d1a0156c54a85b0976ab38c6a64136458ca7c4ee491566d25d0
+
+LDFLAGS+=" -Wl,-no-fatal-warnings"
+
+pre_build() {
+	vsed -i CMakeLists.txt -e 's|plugins/gammaray|lib/qt5/plugins/gammaray|'
+}
+
+GammaRay-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision} ${makedepends}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/cmake
+		vmove usr/lib/qt5/mkspecs
+		# probes should go in main package
+		vmove "usr/lib/libgammaray_[^probe]*.so"
```

Comment:
The libraries are conveniently named. This should be much better

Reply:
^ is not a negative match over string. Please check if results are as expected. You may need to vmove all libgammaray them move back probe.

Reply:
I wasn't sure that it is correct either, but it results in the correct libraries being moved.